doc/api.yml

```diff
@@ -337,13 +337,20 @@ components:
           enum:
             - ARGENT
             - MATERIEL
-            - FLUXARGENT
+            - FLUX_ARGENT
+            - GROUPE_POSSESSION
+            - ACHAT_MATERIEL_AU_COMPTANT
+            - TRANSFERT_ARGENT
```

Comment:
for consistency, if GROUPE_POSSESSION is in the enum list, you should add a groupe_possession attribute to the "PossessionAvecType" and add "possessions" there

Reply:
fix